tests/unit/mock/config/conversion/paloalto_panos/paloalto_basic_converted.txt

```diff
@@ -0,0 +1,20 @@
+set mgt-config users admin phash *
```

Comment:
After a conversion was done, was this config actually pasted onto a true palo alto device.  Just to make sure nothing is converting wrong?

Reply:
I do not have one to test on. If someone has one that would be good. I did a visual compare to the original configs they were based on and they looked good.

Reply:
for what its worth, batfish parsing is complianing about the banner quouting.
```
>>> issues = bf.q.initIssues().answer().frame().iloc[65]
>>> issues
Nodes                                      None
Source_Lines                 [configs/p.cfg:[]]
Type                    Parse warning (redflag)
Details           Improperly-quoted string: "\n
Line_Text                                  None
Parser_Context                             None
Name: 65, dtype: object
```

Reply:
The banner in the converted config is identical to how it shows up in a backup file. As far as I know there is no extra tweaking going on with the backup files, they should be raw from the device/panorama. Can batfish look at a brace format config and does it give the same error?

Reply:
So upon further investigation, it does not appear that you can set a multi-line banner from the CLI (only from the GUI), but the format included is how the backup/set format presents it. Kind of seems like this would be the proper format to keep it in since that's also how golden config is going to compare it?